src/main/resources/swagger.api/self.yaml

```diff
@@ -0,0 +1,65 @@
+openapi: 3.0.0
+info:
+  title: Self integration API
```

Comment:
define proper names and description here

Reply:
changed method name and summary.